src/aws/cis-1.3.0/rules/aws-cis-1.3.0-3.3.ts

```diff
@@ -0,0 +1,111 @@
+  7. The tab shows a list of grants, one row per grant, in the bucket ACL. Each row identifies the grantee and the permissions granted.
+  8. Ensure no rows exists that have the Grantee set to Everyone or the Grantee set to Any Authenticated User.
+  9. If the Edit bucket policy button is present, click it to review the bucket policy.
+  10. Ensure the policy does not contain a Statement having an Effect set to Allow and a Principal set to "*" or {"AWS" : "*"}
+
+  Via CLI:
+
+  1. Get the name of the S3 bucket that CloudTrail is logging to:
+
+          aws cloudtrail describe-trails --query 'trailList[*].S3BucketName'
+
+  2. Ensure the AllUsers principal is not granted privileges to that <bucket> :
+
+          aws s3api get-bucket-acl --bucket <s3_bucket_for_cloudtrail> --query 'Grants[?Grantee.URI== http://acs.amazonaws.com/groups/global/AllUsers ]'
+
+  3. Ensure the AuthenticatedUsers principal is not granted privileges to that <bucket> :
+
+          aws s3api get-bucket-acl --bucket <s3_bucket_for_cloudtrail> --query 'Grants[?Grantee.URI== http://acs.amazonaws.com/groups/global/Authenticated Users ]'
+
+  4. Get the S3 Bucket Policy
+
+          aws s3api get-bucket-policy --bucket <s3_bucket_for_cloudtrail>
+
+  5. Ensure the policy does not contain a Statement having an Effect set to Allow and a Principal set to "*" or {"AWS" : "*"}
+
+  Note: Principal set to "*" or {"AWS" : "*"} allows anonymous access.`,
+  rationale:
+    'Allowing public access to CloudTrail log content may aid an adversary in identifying weaknesses in the affected account`s use or configuration.',
+  remediation: `Perform the following to remove any public access that has been granted to the bucket via an ACL or S3 bucket policy:
+
+  1. Go to Amazon S3 console at https://console.aws.amazon.com/s3/home
+  2. Right-click on the bucket and click Properties
+  3. In the Properties pane, click the Permissions tab.
+  4. The tab shows a list of grants, one row per grant, in the bucket ACL. Each row identifies the grantee and the permissions granted.
+  5. Select the row that grants permission to Everyone or Any Authenticated User
+  6. Uncheck all the permissions granted to Everyone or Any Authenticated User (click x to delete the row).
+  7. Click Save to save the ACL.
+  8. If the Edit bucket policy button is present, click it.
+  9. Remove any Statement having an Effect set to Allow and a Principal set to "*" or {"AWS" : "*"}.`,
+  references: [
+    'CCE-78915-6',
+    'https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html',
+  ],
+  gql: `{
+    queryawsCloudtrail {
+      id
+      arn
+      accountId
+      __typename
+      s3 {
+        policy {
+          statement {
+            effect
+            principal {
+              key
+              value
+            }
+          }
+        }
+      }
+    }
+  }`,
+  resource: 'queryawsCloudtrail[*]',
+  severity: 'medium',
+  conditions: {
```

Comment:
I cant tell, does this check the first parts of the check description?

 `Ensure the AllUsers principal is not granted privileges to that <bucket> `

and

`Ensure the AuthenticatedUsers principal is not granted privileges to that <bucket> `

Reply:
Yes, you're right, I'll check it out. Thanks!

Reply:
@tyler-dunkel  done. This change depends on this PR: https://github.com/cloudgraphdev/cloudgraph-provider-aws/pull/63